src/modules/fetcher/__mocks__/NativeFetcherModule.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import type { FetchResultType, TargetFilePathsType } from '../FetcherModule'
+import { NativeEventEmitter } from 'react-native'
+import moment from 'moment-timezone'
+import { mapValues } from 'lodash/object'
+
+function mockFetchAsync (targetFilePaths: TargetFilePathsType): Promise<FetchResultType> {
+  const fetchResult = mapValues(targetFilePaths, url => ({
+    lastUpdate: moment.tz('20110204', 'UTC').toISOString(),
```

Comment:
You can simplty use `2004-06-14T23:34:30Z` here instead of using moment-timezone